ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -1,104 +1,147 @@
-                                            kind: res.kind,
-                                            namespace: res.namespace,
-                                            group: res.group,
-                                            info: null,
-                                            uid: '',
-                                            resourceVersion: null,
-                                            parentRefs: []
-                                        })
-                                    }
-                                </DropDown>
+                            <div className='columns small-2 xxxlarge-2'>
+                                {res.name}
+                                {res.kind === 'Application' && (
+                                    <Consumer>
+                                        {ctx => (
+                                            <span className='application-details__external_link'>
+                                                <a href={ctx.baseHref + 'applications/' + res.namespace + '/' + res.name} title='Open application'>
+                                                    <i className='fa fa-external-link-alt' />
+                                                </a>
+                                            </span>
+                                        )}
+                                    </Consumer>
+                                )}
+                            </div>
+                            <div className='columns small-1 xxxlarge-1'>{[res.group, res.kind].filter(item => !!item).join('/')}</div>
+                            <div className='columns small-1 xxxlarge-1'>{res.syncWave || '-'}</div>
+                            <div className='columns small-2 xxxlarge-2'>{res.namespace}</div>
+                            {res.kind === 'ReplicaSet' &&
+                                ((getResNode(tree.nodes, nodeKey(res)) as ResourceNode).info || [])
+                                    .filter(tag => !tag.name.includes('Node'))
+                                    .slice(0, 4)
+                                    .map((tag, i) => {
+                                        return (
+                                            <div key={i} className='columns small-1 xxxlarge-1'>
+                                                {tag?.value?.split(':')[1] || '-'}
+                                            </div>
+                                        );
+                                    })}
+
+                            <div className='columns small-2 xxxlarge-2'>
+                                {res.createdAt && (
+                                    <span>
+                                        <Moment fromNow={true} ago={true}>
+                                            {res.createdAt}
+                                        </Moment>
+                                        &nbsp;ago &nbsp; {format(new Date(res.createdAt), 'dd/mm/yyyy')}
```

Comment:
Check the format of this date.  I'm getting 45 for March.

Reply:
Fixed
![Screenshot 2023-03-28 at 6 24 26 PM](https://user-images.githubusercontent.com/11219262/228402464-055f5f7a-d32e-4026-a70e-6b895ea2ed2a.png)